iina/AppData.swift

```diff
@@ -11,7 +11,7 @@ import Cocoa
 struct AppData {
   
   /** time interval to sync play pos */
-  static let getTimeInterval: Double = 0.5
+  static let getTimeInterval: Double = 0.1
```

Comment:
A shorter time interval may consume more energy.

Perhaps it's better to detect the video duration first and set the interval dynamically. What do you think?